code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/woodcutter.dm

```diff
@@ -10,21 +10,17 @@
 /datum/outfit/job/roguetown/adventurer/woodcutter/pre_equip(mob/living/carbon/human/H)
 	..()
 	H.mind.adjust_skillrank(/datum/skill/combat/axes, 3, TRUE) // AXE MEN! GIVE ME SPLINTERS!
-	H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE) 
+	H.mind.adjust_skillrank(/datum/skill/misc/athletics, 2, TRUE) 
 	H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE)
 	H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
 	H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
-	H.mind.adjust_skillrank(/datum/skill/combat/polearms, 1, TRUE) 
 	H.mind.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
-	H.mind.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
-	H.mind.adjust_skillrank(/datum/skill/craft/crafting, 2, TRUE)
-	H.mind.adjust_skillrank(/datum/skill/craft/carpentry, 1, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/misc/climbing, 4, TRUE) // Need to climb to build
+	H.mind.adjust_skillrank(/datum/skill/craft/crafting, 3, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/craft/carpentry, 4, TRUE) 
```

Comment:
Lower carpentry and climbing to 3 please!

Reply:
yes and take off engineering, I forgor

Reply:
I am not agaisnt lowering both skills, but they have those values, as that was what the carpenters used to have in the first place. May I hear the reason you wish them lowered?

Reply:
Because no role should be all mighty. If they wish to specialize they may, but it will cost them.